CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+  - The merger can use the “Squash and merge” option. If they do this, the merger is responsible for cleaning up the commit message according to the previously stated commit message guidance.
+  - The pull request author, after getting the requisite approvals, can reorganize the commits as they see fit (using, for example, git rebase -i) and re-push.
+
+Once the PR is merged, please change the status of the linked issue in [rancher project](https://github.com/rancher/rancher) to "Needs QA". In the following days, the QA team will run tests and process the PR to verify that no regressions were introduced. Please be alert on the issue as the QA team might ask questions related to the PR. 
+
+### Backports
+
+#### RKE code
+
+To understand how backports should be pulled through, RKE project releases and what is included in each release should be described. RKE in general maintains two tracks in parallel which are differentiated by the minor version. Each of these tracks s developped in two different GitHub branches:
+* release/v1.x, (for example [release/v1.5](https://github.com/rancher/rke/tree/release/v1.5)
+* release/v1.(x-1) (for example [release/v1.4](https://github.com/rancher/rke/tree/release/v1.4)
+
+Each track releases an RKE binary. Each release of RKE officially supports a few Kubernetes versions. Some versions are shared by both tracks but normally the track with the higher minor version includes at least one newer Kubernetes version. For example, as it can be observed in [release v1.4.11](https://github.com/rancher/rke/releases/tag/v1.4.11), the following Kubernetes versions are supported:
+* v1.26.9-rancher1-1
+* v1.25.14-rancher1-1
+* v1.24.17-rancher1-1
+* v1.23.16-rancher2-3
+
+whereas [release v1.5.0](https://github.com/rancher/rke/releases/tag/v1.5.0) includes:
+* v1.25.14-rancher1-1
+* v1.26.9-rancher1-1
+* v1.27.6-rancher1-1
+
+If a Kubernetes version is supported, that means we can deploy a cluster with that Kubernetes version using the binary included in that RKE release.
+
+When doing a commit in RKE code, we must carefully think if we want that commit in both tracks and hence backported. Sometimes, there is a reason to no backport it, for example a flag, that we would like to limit to only the newer track. However, that is exceptional and normally commits end up in both branches as a way to reduce the differences between those and simplify RKE maintainability. Therefore, usually we backport a commit between both RKE tracks. In that case, the following steps should be applied:
+
+* Create an Issue in rancher/rancher github project for Project Managers and QA to track the commit. Set the decided milestone for release/v1.x in the issue
```

Comment:
Are we now creating `release/v1.x` milestone under rancher/rancher? We usually create this milestone under rke.

Reply:
If I understood correctly:
* milestones are created in rancher/rancher
* branch `releases/v1.x` is created in rancher/rke project
And they are connected. For example:
* milestone [v2.8-KDM-Nov-2023-patches](https://github.com/rancher/rancher/issues?q=is%3Aopen+is%3Aissue+milestone%3Av2.8-KDM-Nov-2023-patches) in rancher/rancher maps to branch [release/v1.5](https://github.com/rancher/rke/tree/release/v1.5) in rancher/rke
* milestone [v2.7-KDM-Nov-2023-patches](https://github.com/rancher/rancher/issues?q=is%3Aopen+is%3Aissue+milestone%3Av2.7-KDM-Nov-2023-patches) in rancher/rancher maps to branch [release/v1.4](https://github.com/rancher/rke/tree/release/v1.4) in rancher/rke
right?

Reply:
Assuming what I wrote is correct, let me add one extra sentence to explain this. I can see how it could be confusing for the reader